website/docs/guides/2.0-beta.html.markdown

```diff
@@ -1,9 +1,279 @@
 ---
 layout: "azurerm"
-page_title: "Azure Resource Manager: 2.0 Beta"
+page_title: "Azure Resource Manager: Opting into the 2.0 Beta"
 description: |-
-    Azure Resource Manager: 2.0 Beta
+    Azure Resource Manager: Opting into Beta Functionality coming in version 2.0 of the Azure Provider
 
 ---
 
-# Coming Soon
+# Opting into Beta Functionality coming in version 2.0 of the Azure Provider
+
+Version 2.0 of the Azure Provider will introduce several new features:
```

Comment:
Should we also call out that we'll be removed all deprecated properties and resources?

Reply:
cross-linked to the 2.0 upgrade doc